phpBB/phpbb/filesystem/helper.php

```diff
@@ -233,7 +233,7 @@ public static function resolve_path($path, $prefix = '', $absolute = false, $ret
 			$path = str_replace(DIRECTORY_SEPARATOR, '/', $path);
 		}
 
-		trim($path, '/');
+		//$path = trim($path, '/'); // TODO: Check this
```

Comment:
I don't see a reason to remove this. It could prevent potentially worry-some absolute paths.

Reply:
```php
trim($path, '/');```

Was doing nothing, because the trimmed value was being returned to nowhere. I just modified it and commented it, because it fails in windows if I don't remember wrong, it has to be investigated, but when doing that I haven't modified the current behavior.